provider/image.go

```diff
@@ -849,8 +850,14 @@ func configureDockerClientInner(configs map[string]string, host string) (*client
 	} else {
 		// No TLS certificate material provided, create an http client
 		if host != "" {
+			var sshopts []string
+			if opts, ok := configs["sshOpts"]; ok {
+				err = json.Unmarshal([]byte(opts), &sshopts)
```

Comment:
Nit: This keeps variables local.
```suggestion
				err := json.Unmarshal([]byte(opts), &sshopts)
```